src/main/compatibility-check.ts

```diff
@@ -6,8 +7,14 @@ import rimraf from 'rimraf'
 import {app, dialog} from 'electron'
 import {DATADIR_VERSION, COMPATIBLE_VERSIONS} from '../shared/version'
 import {config} from '../config/main'
```

Comment:
Config value should really be imported only in `main.ts` file, in every other place it should be received as function argument.